pom.xml

```diff
@@ -370,7 +370,12 @@ under the License.
     <dependency>
       <groupId>org.codehaus.plexus</groupId>
       <artifactId>plexus-utils</artifactId>
-      <version>3.5.1</version>
+      <version>4.0.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.codehaus.plexus</groupId>
+      <artifactId>plexus-xml</artifactId>
+      <version>4.0.0</version>
```

Comment:
must be 3.0.0

Reply:
Why actually? This piece of information should be documented somewhere...

Reply:
https://github.com/codehaus-plexus/plexus-xml/commit/7bfeb09596b87f4a28add4699bbd663fe6dcbd6e

Reply:
I see, then this needs to be checked with all components updated to 40.

Reply:
Yes I'm reviewing it. Dependabot helps with notification here.